default/python/AI/empire/survey_universe.py

```diff
@@ -63,8 +63,6 @@ def survey_universe():
             planet = universe.getPlanet(pid)
             if not planet:
                 continue
-            if pid in aistate.colonisablePlanetIDs:
-                empire_has_qualifying_planet = True
```

Comment:
Just having a matching planet in the list doesn't mean we'll settle it soon.
This should be completely revised. Having a gas giant no longer implies we want to build a GGG, especially when playing Sly. But that is for another PR.